poker/scraper/table_setup_actions_and_signals.py

```diff
@@ -261,11 +260,6 @@ def _connect_cards_with_save_slot(self):
         button_show_property = getattr(self.ui, 'covered_card_show')
         button_show_property.clicked.connect(lambda state: self.load_image('covered_card'))
 
-    def _connect_save_max_players_with_save_slot(self):
```

Comment:
if this is removed, how does it save any changes?

Reply:
I don't know how to cite a code line here. But in Line 101:
`self.ui.max_players.currentIndexChanged[int].connect(lambda: self._save_max_players())`
there is already a connect for a changed index. The value has been saved twice, when selected Index changed.